lib/logstash/plugin_mixins/rabbitmq_connection.rb

```diff
@@ -215,7 +215,7 @@ def connect
         end
 
         channel = connection.create_channel
-        @logger.info("Connected to RabbitMQ at #{rabbitmq_settings[:host]}")
+        @logger.info("Connected to RabbitMQ", :url => connection_url(connection))
```

Comment:
This would log sensitive password information. You could try instantiating it a [SafeURI](https://github.com/elastic/logstash/blob/master/logstash-core/lib/logstash/util/safe_uri.rb) instead, which has a `to_s` method that doesn't log the password.

For this code, however, the simplest thing would probably be to not use the `connection_url` method and just build a string with everything but the password.